definitions/vertical_scaling.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Vertical Scaling
+status: feedback appreciated
+category: concept
+---
+## Canary Deployment
+
+### What it is
+
+
+### Problem it Addresses
+[Scalability](https://github.com/himanshu007-creator/glossary/blob/main/definitions/scalability.md) is essential in that it contributes to competitiveness, efficiency, reputation and quality. Vertical scaling is on one of the two kinds (Vertical,Horizontal) of scalability.
```

Comment:
Looks good unfortunately the link is to your fork. do you mind updating it to track main?

Reply:
Also do you want to add something to talk about the reason something might need to be scaled up instead of out?